.github/workflows/push-zac-container.yml

```diff
@@ -13,7 +13,9 @@ jobs:
       - name: Login to Docker Hub
         uses: docker/login-action@v1
         with:
-          username: ${{ secrets.DOCKER_HUB_API_USER }}
+          username: ${{ vars.DOCKER_HUB_API_USER || secrets.DOCKER_HUB_API_USER }}
```

Comment:
what's the usefulness of addding `vars.DOCKER_HUB_API_USER`? same with ZAC_IMAGE_REPO? i'm missing why it's changing?

Reply:
This allows a clone of the official repo to test the CI.

Reply:
to prevent further litigation - would you rever this? thx